test/e2e/density.go

```diff
@@ -210,6 +214,7 @@ var _ = Describe("Density", func() {
 				PodStatusFile:        fileHndl,
 				Replicas:             totalPods,
 				MaxContainerFailures: &MaxContainerFailures,
+				Timeout:              time.Duration(math.Ceil(float64(itArg.podsPerNode)/minPodStartupThroughput)) * time.Minute,
```

Comment:
Please don't change it - it's not connected only to kubelet throughput, but also it's related to scheduler and rest of the system. And we definitely not have 15 pods/s throughput on scheduler for large clusters right now.